game/scripts/npc/abilities/legion_commander_moment_of_courage.txt

```diff
@@ -13,7 +13,7 @@
     "MaxLevel"                                            "6"
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "0.8" //OAA
+    "AbilityCooldown"                                     "1.5 1.2 0.9 0.6 0.3 0.3" //OAA
```

Comment:
Set this to 0.3 and leave the proc chance scaling like it was. (Its a similar change Axe got for his counter helix)

Reply:
We can try this but the proc chance is going to have to be lower then it currently is.

Reply:
Legion's passive is the only thing that gives her carry potential (beside Duel, Duel is overbuffed already).

Reply:
I cannot see this being balanced but its worth a try.